<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1083</width>
    <height>620</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>1</number>
    </property>
    <property name="topMargin">
     <number>1</number>
    </property>
    <property name="rightMargin">
     <number>1</number>
    </property>
    <property name="bottomMargin">
     <number>1</number>
    </property>
    <item>
     <widget class="QPlainTextEdit" name="TextEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1083</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件(&amp;F）</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="action_O"/>
    <addaction name="action_S"/>
    <addaction name="action_A"/>
    <addaction name="separator"/>
    <addaction name="action_X"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>编辑(&amp;E)</string>
    </property>
    <addaction name="action_Z"/>
    <addaction name="action_Shift_Z"/>
    <addaction name="action_X_2"/>
    <addaction name="action_C"/>
    <addaction name="action_V"/>
    <addaction name="action_F"/>
    <addaction name="action_R"/>
    <addaction name="separator"/>
    <addaction name="action_A_2"/>
   </widget>
   <widget class="QMenu" name="menu_O">
    <property name="title">
     <string>格式(&amp;O）</string>
    </property>
    <addaction name="action_W"/>
    <addaction name="action_L"/>
    <addaction name="action_F_2"/>
    <addaction name="action_3"/>
    <addaction name="separator"/>
    <addaction name="action_4"/>
    <addaction name="action_5"/>
   </widget>
   <widget class="QMenu" name="menu_V">
    <property name="title">
     <string>查看(&amp;V）</string>
    </property>
    <addaction name="action_L_2"/>
    <addaction name="action_7"/>
   </widget>
   <widget class="QMenu" name="menu_H">
    <property name="title">
     <string>帮助(&amp;H)</string>
    </property>
    <addaction name="action_A_3"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_O"/>
   <addaction name="menu_V"/>
   <addaction name="menu_H"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_O"/>
   <addaction name="actionNew"/>
   <addaction name="action_S"/>
   <addaction name="action_A"/>
   <addaction name="separator"/>
   <addaction name="action_Z"/>
   <addaction name="action_Shift_Z"/>
   <addaction name="action_X_2"/>
   <addaction name="action_C"/>
   <addaction name="action_V"/>
   <addaction name="action_F"/>
   <addaction name="action_R"/>
   <addaction name="separator"/>
   <addaction name="action_W"/>
   <addaction name="action_F_2"/>
   <addaction name="separator"/>
   <addaction name="action_L"/>
   <addaction name="action_L_2"/>
   <addaction name="action_7"/>
   <addaction name="separator"/>
  </widget>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="notepad.qrc">
     <normaloff>:/image/invoice-text-plus-outline.png</normaloff>:/image/invoice-text-plus-outline.png</iconset>
   </property>
   <property name="text">
    <string>新建(&amp;N）</string>
   </property>
   <property name="toolTip">
    <string>新建</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="action_O">
   <property name="icon">
    <iconset resource="notepad.qrc">
     <normaloff>:/image/folder-open-outline.png</normaloff>:/image/folder-open-outline.png</iconset>
   </property>
   <property name="text">
    <string>打开(&amp;O）</string>
   </property>
   <property name="toolTip">
    <string>打开</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="action_S">
   <property name="icon">
    <iconset resource="notepad.qrc">
     <normaloff>:/image/content-save-check-outline.png</normaloff>:/image/content-save-check-outline.png</iconset>
   </property>
   <property name="text">
    <string>保存(&amp;S）</string>
   </property>
   <property name="toolTip">
    <string>保存</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="action_A">
   <property name="icon">
    <iconset resource="notepad.qrc">
     <normaloff>:/image/content-save-move.png</normaloff>:/image/content-save-move.png</iconset>
   </property>
   <property name="text">
    <string>另存为(A）</string>
   </property>
   <property name="toolTip">
    <string>另存为</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="action_X">
   <property name="text">
    <string>退出(E)</string>
   </property>
   <property name="toolTip">
    <string>退出</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="action_Z">
   <property name="icon">
    <iconset resource="notepad.qrc">
     <normaloff>:/image/reply-outline.png</normaloff>:/image/reply-outline.png</iconset>
   </property>
   <property name="text">
    <string>撤销(U)</string>
   </property>
   <property name="toolTip">
    <string>撤销</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="action_Shift_Z">
   <property name="icon">
    <iconset resource="notepad.qrc">
     <normaloff>:/image/share-outline.png</normaloff>:/image/share-outline.png</iconset>
   </property>
   <property name="text">
    <string>恢复(）</string>
   </property>
   <property name="toolTip">
    <string>恢复</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Z</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="action_X_2">
   <property name="icon">
    <iconset resource="notepad.qrc">
     <normaloff>:/image/content-cut.png</normaloff>:/image/content-cut.png</iconset>
   </property>
   <property name="text">
    <string>剪切(T）</string>
   </property>
   <property name="toolTip">
    <string>剪切</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="action_C">
   <property name="icon">
    <iconset resource="notepad.qrc">
     <normaloff>:/image/content-copy.png</normaloff>:/image/content-copy.png</iconset>
   </property>
   <property name="text">
    <string>复制(C）</string>
   </property>
   <property name="toolTip">
    <string>复制</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="action_V">
   <property name="icon">
    <iconset resource="notepad.qrc">
     <normaloff>:/image/content-paste.png</normaloff>:/image/content-paste.png</iconset>
   </property>
   <property name="text">
    <string>粘贴(P)</string>
   </property>
   <property name="toolTip">
    <string>粘贴</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="action_F">
   <property name="icon">
    <iconset resource="notepad.qrc">
     <normaloff>:/image/file-find-outline.png</normaloff>:/image/file-find-outline.png</iconset>
   </property>
   <property name="text">
    <string>查找(F）</string>
   </property>
   <property name="toolTip">
    <string>查找</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="action_R">
   <property name="icon">
    <iconset resource="notepad.qrc">
     <normaloff>:/image/replace.png</normaloff>:/image/replace.png</iconset>
   </property>
   <property name="text">
    <string>替换(R)</string>
   </property>
   <property name="toolTip">
    <string>替换</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="action_A_2">
   <property name="text">
    <string>全选(&amp;A）</string>
   </property>
   <property name="toolTip">
    <string>全选</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="action_W">
   <property name="icon">
    <iconset resource="notepad.qrc">
     <normaloff>:/image/keyboard-return.png</normaloff>:/image/keyboard-return.png</iconset>
   </property>
   <property name="text">
    <string>自动换行(W）</string>
   </property>
   <property name="toolTip">
    <string>自动换行</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="action_L">
   <property name="icon">
    <iconset resource="notepad.qrc">
     <normaloff>:/image/format-list-numbered.png</normaloff>:/image/format-list-numbered.png</iconset>
   </property>
   <property name="text">
    <string>显示行号(L)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="action_F_2">
   <property name="icon">
    <iconset resource="notepad.qrc">
     <normaloff>:/image/format-size.png</normaloff>:/image/format-size.png</iconset>
   </property>
   <property name="text">
    <string>字体(F）</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>字体颜色</string>
   </property>
   <property name="toolTip">
    <string>字体颜色</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>字体背景色</string>
   </property>
   <property name="toolTip">
    <string>字体背景色</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="action_5">
   <property name="text">
    <string>编辑器背景色</string>
   </property>
   <property name="toolTip">
    <string>编辑器背景色</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="action_L_2">
   <property name="icon">
    <iconset resource="notepad.qrc">
     <normaloff>:/image/tools.png</normaloff>:/image/tools.png</iconset>
   </property>
   <property name="text">
    <string>工具栏(L）</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="action_7">
   <property name="icon">
    <iconset resource="notepad.qrc">
     <normaloff>:/image/list-status.png</normaloff>:/image/list-status.png</iconset>
   </property>
   <property name="text">
    <string>状态栏</string>
   </property>
   <property name="toolTip">
    <string>状态栏</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="action_A_3">
   <property name="text">
    <string>关于(A）</string>
   </property>
   <property name="toolTip">
    <string>关于(A）</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
 </widget>
 <resources>
  <include location="notepad.qrc"/>
 </resources>
 <connections/>
</ui>
